### PR TITLE
first cypress test

### PR DESCRIPTION
**Pull request by edopore**
Screenshot first cypress test
imagen

### CODE REVIEW

File: cypress/e2e/google.cy.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe("This is my first cypress test", () => {
+    it("should have a title", () => {
+      cy.visit("https://www.google.com/");
+      cy.title().should("eq", "Google");
+    });
+  });
+
```
Comment (veronicatofino) [inline cypress/e2e/google.cy.js:7]: EOF

Comment (holgiosalos) [reply cypress/e2e/google.cy.js:7]: probably line 7 has whitespaces, remove them and you will fix the EOF issue

Comment (kliver98) [reply cypress/e2e/google.cy.js:7]: Remove space you have in line 7
image
image

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (holgiosalos) [inline cypress/support/commands.js:25]: EOF

File: cypress/support/e2e.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (holgiosalos) [inline cypress/support/e2e.js:20]: EOF

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**